README.md

```diff
@@ -0,0 +1,19 @@
+# Development
+
+1. `echo DEBUG=True >> .env`
+2. `virtualenv env`
+3. `source env/bin/activate`
+4. `pip install -r requirements.txt`
+5. `npm install`
+6. In another terminal, set up and run the database
+    1. Install postgres
+        * Mac: `brew install postgres`
+    2. Create a new user with name *distributionviewer* and password *distributionviewer*
+        * Mac: `createuser --pwprompt distributionviewer`
+    3. Create a new database named *distributionviewer* that the
+       *distributionviewer* user has read/write access to
+        * Mac: `createdb -Odistributionviewer -Eutf8 distributionviewer`
+    4. Run postgres
+        * Mac: `postgres -D /usr/local/var/postgres`
```

Comment:
Ah, I like that approach. Didn't know `brew` could do that. I'll move this up, too.